### DISCUSSION TITLE
User Manual for the THC Isolated ADC available now.

### DISCUSSION THREAD

**Discussion by phil-barrett** [grblHAL/Plugin_plasma#6] [General]
V1.0 of the [THC Isolated ADC manual](https://github.com/phil-barrett/ADCI_1/blob/main/user%20manual.pdf) is now available.

Feedback, suggestions or corrections are greatly appreciated.  Either here or on the [THC Isolated ADC github page](https://github.com/phil-barrett/ADCI_1).

Also, I still have a few boards available.

**Comment by Toutoune86** [comment]
Hello, 
I write one month later, do you still have these THC boards ? Do you sell the breakboards as well, and hopefully thus ship them together ?

**Reply by phil-barrett** [reply]
Yes, I have a couple. $20.  And yes, I still have the [breakout boards](https://www.grbl.org/grblhal-breakout-board).   I can ship together, no extra charge.  Make a note in the tindie purchase (special instructions) - say ADC and Toutoune86 so I can connect the order to your post here. 

You can pay the $20 via paypal separately from the tindie transaction (I haven't put it as a tinde product yet).  I will send instructions after the tindie order. Should not take much extra time but ping me here as well - tindie has been recently having some email notification problems. 

Phil

**Reply by DBS3262** [reply]
Hi Phil,
Do you have any of the THC ADC boards still available? I would love to purchase one along with the breakout board you have on tindie. Ive been looking on tindie but have only successfully found the breakout board.

Thanks, 

Dana

**Reply by phil-barrett** [reply]
Yes, I just got a small run of the final version in.  I've had a lot of work to do lately but will be putting the ADC boards up on Tindie.  They will be priced at $29.99 USD through Tindie. (The ADC chip is annoyingly expensive for what it does.) I still have a number of things left to get the board up on Tindie so I'll sell direct to you for $20 USD. US Shipping is$5.50, contact me if non-US shipping. If you order a T41U5XBB on Tindie, I will ship them together and save you the postage.  You will need to contact me on Tindie if you want to do that so I can give you payment instructions for the ADC.

Phil

**Comment by Toutoune86** [comment]
Ok, thanks for the informations.
I live in France, could you ship the boards here ? How much would it be for
the shipment ?
I hope it would not be too much a pain to send abroad.

Best regards,

Tristan Cousin.

**Reply by phil-barrett** [reply]
The shipping costs are shown on the [tindie site](https://www.tindie.com/products/philba/grblhal-bob-unkit-for-teensy-41-t41u5xbb/). Shipping to France is a little less than $20 USD.  No additional shipping cost for the ADC.

I ship internationally so it is very easy for me.

**Comment by Toutoune86** [comment]
Hello again
and Happy new year !

I have spent some times reading your manual and some of you blog posts to
understand better the process of assembling your T41U5XBB, loading its
Teensy with the apropriate firmware, and running it. The aim is to foresee
some possible issues or incompatibility between your device on one side and
my needs and situation. I guess you do not earn a lot from each sale
compared to what I would have to pay, considering your expenses, shipping
fees, TAV, etc. I will therefore try to be as short as possible in my
questions and thank you warmly for whatever replies you may take the time
to give me. These are the questions the answers of which I was not able to
reply or understand  fully through your documentation and my knowledge.

My situtation is this one:
I plan to use my CNC mainly  for plasma cutting, with 3 cartesian axis and
a 4th rotary axis. I currently still have some EMI issues remaining in
spite of various shielding and grounding tasks. I expect especially your
electronic design to relieve this problem. I guess therefore the ethernet
version is better than the USB version, I think USB cable was the cause of
most o y issues (And yes, I fully agree, EMI is so invisible and
mysterious, it often looks like vaudoo, I used to call it shamanic until I
read you)
I have no Internet connection in my workshop, therfore no router yet. I
just have one motor and driver for my Y axis, on which my X axis is in
hangover. It may be sufficient for a plasma use, not for possible addons
like a routing  spindle. So I may later install a 2nd mirrored Y motor and
hence need a 5th axis control, should I want to make X an Y square.

My main questions are about the ehternet feature cable :

   - In the manual (page4) you say there may be two versions one with
   ethernet, another one with USB, which is not the case yet. I would like to
   be sure manual is up to date and the breakoutboard is ethernet compatible,
   even though it is called on Tindie a "t41*U*xbb".


   - It may look like a stupid question, but I wonder if the connection
   with the router or ethernet switch would work even whitout internet
   connection.


   - Also, I fear the the router or switch, or even the use of an extra
   ethernet cable between the router and the computer may be as well another
   exposure to IME and corrupt the connection. May be the TCP/IP protocol
   itself prevents from such risk ? I would be glad to have your feedback
   regarding the experiences you had or heard about regarding this
   configuration


   - I was considering as well to use a single and direct ethernet
   connection to my PC (pages 14 and 15), even though you say it may be more
   complex. But I just have one network interface connector in my PC. So I
   should in this case  use an USB adpapter, which is pretty sensitive to EMI,
   I guess, no ? I have not understood if this aditionnal connector was just
   plugged for a temporary use, ie to set the  IP address, gateway and mask,
   or whether it would have to remain and therefore pose a remainng EMI
   vulnerability risk.


   - I did not undertsand your advice page 14 "2. Secure a cross-over
   Ethernet cable. It may be unnecessary but it won't hurt." Does it mean we
   may rather need a straight cable and can not know it yet until trying ?
   does "secure" mean we may ned to have one axtra available ?


   - If the the Ethernet may not work properly due to EMI with T41U5XBB, do
   you think the Pico's wifi may be a relevant, noise-proof solution ?

My last questions regard the THC plugin. I am just autodidact and in no way
engineer in electronics and do not understand  the I2C plugging.

   - How does it work ? I see on the THC board that tere are the same pins
   than those dedicated to I2C on the breakout board ( Halt, +V, St, Sd Sc,
   Ground).  Is it enough, as for hardware connection achivement, to connect
   each end of 6 wires to the pin of the same name in each board ? (then I
   would recommend to add it to the manual, i may not be so obvious for other
   people too)


   - And what does ADC mean in this case ? Analog/Digital Converter?


I thank you both for your work in general and for your replies. I am
looking forward to finish my plasma CNC, and would be glad later if I can,
to give some THC plugin feedbacks.

Best regards,


Le sam. 16 déc. 2023 à 18:23, phil-barrett ***@***.***> a
écrit :

> The shipping costs are shown on the tindie site
> <https://www.tindie.com/products/philba/grblhal-bob-unkit-for-teensy-41-t41u5xbb/>.
> Shipping to France is a little less than $20 USD. No additional shipping
> cost for the ADC.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/grblHAL/Plugin_plasma/discussions/6#discussioncomment-7872050>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGQ464FQ7YWUSM2XQT6NQEDYJXKIJAVCNFSM6AAAAAA7D3MRRGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TQNZSGA2TA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Tristan Cousin
06 45 17 14 03
https://tricyclos.wordpress.com : Mécanique vélo, vélo-cargo et motorisation

**Reply by phil-barrett** [reply]
I'm running out sorry for the terse replies. I will be happy to elucidate further.

ADC = analog to digital
Any ethernet cable will work (crossover is not needed, I need to update the manual, sorry for the confusion)
There is only one board - naming convention happened before I decided to make one board work for both USB and Ethernet.
The plasma ADC plugs into the I2C connector on the board.
If you follow [this guide](https://www.grbl.org/single-post/ethernet-without-routers-for-grblhal), you won't need a router or switch - direct connect from PC to T41U5XBB ethernet port.

**Comment by DBS3262** [comment]
Looks like I’m going to get the ADC only for now. I wasn’t sure my board had the required connection to work, but I’ve since verified I have the i2C.  My shipping info is: Dana Smith 11190 Bonham Ranch Rd Dripping Springs, TX 78620. How and where would you like me to pay ? I think I’d like to buy 2 ADCs. With shippingThanks again!DanaSent from my iPhoneOn Oct 16, 2024, at 2:20 PM, phil-barrett ***@***.***> wrote:﻿
Yes, I just got a small run of the final version in.  I've had a lot of work to do lately but will be putting the ADC boards up on Tindie.  They will be priced at $29.99 USD through Tindie. (The ADC chip is annoyingly expensive for what it does.) I still have a number of things left to get the board up on Tindie so I'll sell direct to you for $20 USD. US Shipping is$5.50, contact me if non-US shipping. If you order a T41U5XBB on Tindie, I will ship them together and save you the postage.  You will need to contact me on Tindie if you want to do that so I can give you payment instructions for the ADC.
Phil

**Reply by phil-barrett** [reply]
Either put your email address here or connect to me via the Tindie page for the Teensy board.  I don't want to put my email address on github and don't recommend others do that but it's your choice (wish they had a secure messaging option).
